### DISCUSSION TITLE
20240307 Weekly Development Meeting

### DISCUSSION THREAD

**Discussion by josecelano** [nautilus-cyberneering/.github#85] [Meetings]
- Time:  9:30 in Canary Islands (WET (UTC+0)).
- Time: 10:30 in Vaduz (CET (UTC+1)).

Location:
<https://meeting.blockfinance-eco.li/NautilusCyberneeringWeeklyDevMeeting>

Expected duration: 2 to 4 hours with breaks

Communicated attendants:

- CGarnham(@da2ce7)  - 
- JCelano(@josecelano) - x
- CBosse(@cgbosse) - x
- MMelchor(@mario-nt) - x
- GByrne(@grmbyrn) - x
- MVanDijke(@WarmBeer) - 

-------------------------------------------------------------

## Agenda:

1. Everyone will take their turn and:
- Go over his notes added as comments for the meeting which will have been added in advance to give an update of all the work done during the previous week indicating.
	- The issues worked on
	- The progress made on them
	- Other things done
- Inform of any faced challenges and request help if necessary to work on specific issues.
- Highlight the encountered difficulties, especially if there is a risk of them causing delays or affecting end product's quality.
- Communicate what will be the next week's tasks.
- List any new topics for further discussion in an additional Technical meeting that he noted for future discussion in comments to this discussion either before the meeting or after his turn so that they get considered.

During the meeting every team member will take their turns and follow this sequence.
The topics for further discussion will be noted in advance by each the team member and to the meetings agenda.

2. At the end of the meeting if there any new technical topics to be discussed further it will be decided to
set another meeting date for the discussion of these or to hold the meeting hereafter.

Also the persons to be part of the meeting will be decided on so that the others will be free to continue their own work.

**Comment by josecelano** [comment]
### Torrust Index

Issues:

- https://github.com/torrust/torrust-index/issues/301

Store all fields in `Torrent` struct in database:

- https://github.com/torrust/torrust-index/issues/284
  - https://github.com/torrust/torrust-index/issues/438
  - https://github.com/torrust/torrust-index/issues/437

Canonical InfoHash Group:

- https://github.com/torrust/torrust-index/issues/275
  - https://github.com/torrust/torrust-index/issues/294
  - https://github.com/torrust/torrust-index/issues/290
  - https://github.com/torrust/torrust-index/issues/283

New issues:

- https://github.com/torrust/torrust-index/issues/507

### Next

Continue with issues related to Canonical InfoHash in the Index GUI:

- https://github.com/torrust/torrust-index-gui/issues/496
  - https://github.com/torrust/torrust-index-gui/issues/279
  - https://github.com/torrust/torrust-index-gui/issues/242

**Comment by grmbyrn** [comment]
## Torrust Website
* [fix some ESLint warnings #145](https://github.com/torrust/torrust-website/pull/145)
* [fix ESLint bugs #146](https://github.com/torrust/torrust-website/pull/146)
* [fix svelte check warnings #147](https://github.com/torrust/torrust-website/pull/147)
* [change index naming and misc #148](https://github.com/torrust/torrust-website/pull/148)

## SvelteKit Practice
* [BMI Calculator](https://github.com/grmbyrn/bmi-calculator)
* [Art Gallery Website](https://github.com/grmbyrn/art-gallery-website)

## Research
* Claude3 AI and [Poe AI](https://poe.com/)

**Comment by mario-nt** [comment]
### GUI

- https://github.com/torrust/torrust-index-gui/issues/228

### Index
	
- https://github.com/torrust/torrust-index/issues/448
- https://github.com/torrust/torrust-index-gui/issues/401	

### Discussions

- https://github.com/torrust/torrust-index/issues/507

### Training/Research

- How to read files from server (fetch API, File system API...)
- Readable streams
- Tidy first? book

### New Issues

- https://github.com/torrust/torrust-index-gui/issues/493
